dadi/lib/view/index.js

```diff
@@ -28,40 +27,48 @@ View.prototype.setData = function (data) {
 }
 
 View.prototype.render = function (done) {
-  if (this.json) {
-    // Return the raw data
-    return done(null, this.data)
-  } else {
-    let templateData = _.extend(
-      {
-        host: this.page.hostKey
-      },
-      this.data
-    )
+  // Return the raw data
+  if (this.json) return done(null, this.data)
 
-    this.template
-      .render(templateData, {
-        keepWhitespace: this.page.keepWhitespace
-      })
-      .then(output => {
-        let err = null
+  // Send the templated page
+  let templateData = Object.assign(this.data, {
+    host: this.page.hostKey
+  })
 
-        if (this.page.beautify) {
+  this.template
+    .render(templateData, {
+      keepWhitespace: this.page.keepWhitespace
+    })
+    .then(output => {
+      let err = null
```

Comment:
This variable doesn't seem to ever be changed. You're returning an error on L60 but it comes from L59. Could we remove this initialisation and just do `return done(null, output)` on L65?